language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/workspace/BallerinaWorkspaceManager.java

```diff
@@ -236,18 +236,20 @@ public void didOpen(Path filePath, DidOpenTextDocumentParams params) throws Work
         }
 
         Project project = projectPair.project();
-        if (filePath.equals(project.sourceRoot().resolve(ProjectConstants.BALLERINA_TOML))) {
-            // create or update Ballerina.toml
-            updateBallerinaToml(params.getTextDocument().getText(), projectPair);
```

Comment:
We are not going to allow Ballerina.toml updates?

Reply:
Upon didOpen we are not going to update the content since the project creation is done in the first step.

Reply:
@Kishanthan we are shipping .toml updates. This use-case is already handled in the early methods calls.